src/jaxsim/api/model.py

```diff
@@ -2073,13 +2076,16 @@ def step(
 
         # Compute the 6D forces W_f ∈ ℝ^{n_L × 6} applied to links due to contact
         # with the terrain.
-        W_f_L_terrain = js.contact_model.link_contact_forces(
+        W_f_L_terrain, old_contact_state = js.contact.link_contact_forces(
```

Comment:
I would add a comment here explaining why we are calling a variable "old contact state" since could be tricky to recall in the near future